### ISSUE TITLE
httr update to 1.0.0 breaks tbl()

### ISSUE THREAD

**Issue by akhmed1** [hadley/bigrquery#54]
Hello,

Recent update of httr to [1.0.0](https://github.com/hadley/httr/commit/485d387daffcd76aca454fad74b7b16bafd30acd) switched `httr` to `curl` instead of `RCurl`. 

This update breaks the basic functionality of `bigrquery` at the very least with respect to `tbl` function. In order to confirm, please update to `httr 1.0.0` and run the following to confirm:
``` r
library(dplyr)
library(bigrquery)

BILLING_PROJECT <- "<YOUR_BILLING_PROJECT>"
BILLING_DATASET <- "<DATASET_INSIDE_THAT_PROJECT>"

bq <- src_bigquery(BILLING_PROJECT, BILLING_DATASET)

# Upload the test data frame to your project
df <- data.frame(a = 1:100, b = 101:200)
httr_issue <- copy_to(bq, df, name = "httr_issue")

httr_issue2 <- bq %>% tbl("httr_issue")
```
which will produce the following error if `httr` is version 1.0.0:
``` r
Error in curl::curl_escape(names(url$query)) : 
  `url` must be a character vector.
```
and will work correctly if `httr` is `0.6.1`.

The issue is caused by difference in behavior of curlEscape with NULLs for `curl` vs `RCurl`. Note that `curl::curl_escape(NULL)` results in an error while `RCurl::curlEscape` produces an empty vector.
``` r
> curl::curl_escape(NULL)
Error in curl::curl_escape(NULL) :
    `url` must be a character vector.
> RCurl::curlEscape(NULL)
character(0)
```

As of now, the quick fix is to downgrade `httr` back to `v0.6.1`:
``` r
devtools::install_github("hadley/httr@v0.6.1")
```

**Comment by craigcitro** [comment]
this seems to be fixed by jeroenooms/curl#32, which @jeroenooms managed to release almost instantly. :wink:

**Comment by akhmed1** [comment]
Thanks! It fixes the issue and it is already on CRAN! This is amazing :smile:
